schedule-view/dist/index.html

```diff
@@ -1,15 +1,182 @@
+                    id="Path"></path>
+                  <path
+                    d="M73.502,428.16175 C76.7872,431.43515 81.2403,433.27505 85.881,433.27505 C90.5177,433.27505 94.9708,431.43525 98.256,428.16175 L131.248,395.14575 L131.251907,395.14575 C135.544907,390.70435 137.177707,384.32575 135.548807,378.36475 C133.916007,372.40385 129.263607,367.74375 123.302807,366.10675 C117.345807,364.47005 110.962807,366.098937 106.517807,370.38795 L73.501807,403.45395 C70.232307,406.73125 68.392407,411.17665 68.392407,415.80595 C68.392407,420.43875 70.232207,424.88405 73.501807,428.16095 L73.502,428.16175 Z"
+                    id="Path"></path>
+                  <path
+                    d="M0,250.83175 C0,255.47235 1.8438,259.92555 5.125,263.20675 C8.4062,266.48795 12.8555,268.33175 17.5,268.33175 L64.164,268.33175 C70.4179,268.33175 76.195,264.99585 79.32,259.58175 C82.445,254.16765 82.445,247.49575 79.32,242.08175 C76.195,236.66775 70.4177,233.33175 64.164,233.33175 L17.5,233.33175 C12.8555,233.33175 8.4062,235.17555 5.125,238.45675 C1.8438,241.73795 0,246.19115 0,250.83175 Z"
+                    id="Path"></path>
+                  <path
+                    d="M131.202,131.22175 C134.4872,127.93655 136.3309,123.48735 136.3309,118.84275 C136.3309,114.19815 134.4871,109.74505 131.202,106.46375 L98.206,73.49475 C93.7216,69.48305 87.483,68.05725 81.702,69.72525 C75.9168,71.38935 71.393,75.91275 69.729,81.69825 C68.061,87.47945 69.48681,93.71825 73.4985,98.20225 L106.4675,131.22225 C109.7527,134.49175 114.198,136.32775 118.8345,136.32775 C123.4712,136.32775 127.9165,134.49185 131.2015,131.22225 L131.202,131.22175 Z"
+                    id="Path"></path>
+                </g>
+              </g>
+            </g>
+          </svg>
+        </a>
+      </div>
+
+    </div>
+  </header>
+
   <body>
     <div id="app"></div>
     
+
+    <footer class="container is-fullwidth is-shallow">
+      <div class="footer-nav">
+        <div class="columns">
+          <div class="column is-one-quarter">
+            <p>
+              <a href="https://www.fermyon.com">
+                <img src="/static/image/logo-badge.png" alt="Fermyon" />
+              </a>
+            </p>
+          </div>
+          <div class="column">
+            <nav class="navbar" role="navigation" aria-label="footer navigation">
+              <div class="navbar-end has-text-right">
+                <div class="navbar-item"><a href="https://discord.gg/AAFNfS7NGf">Discord</a></div>
+                <div class="navbar-item"><a href="https://twitter.com/fermyontech">Twitter</a></div>
+                <div class="navbar-item"><a href="https://github.com/fermyon/developer">GitHub</a></div>
```

Comment:
might want to change the GitHub link to "https://github.com/fermyon/spin"

Reply:
Ah yes!